src/components/Image/Image.tsx

```diff
@@ -1,49 +1,73 @@
-import { PropsWithChildren, useRef, useState } from "react";
+import React, {
+  PropsWithChildren,
+  useImperativeHandle,
+  useRef,
+  useState,
+} from "react";
 import ContentLoader from "components/ContentLoader";
 import ExtLink from "components/ExtLink";
 import ImagePlaceholder from "./ImagePlaceholder";
 
-export type ImageProps = {
-  src?: string;
+export type ImageProps = React.ImgHTMLAttributes<HTMLImageElement> & {
```

Comment:
Merged `ImageProps` and `Props` as it makes using `Image` component much more similar to using plain `img`, making it more intuitive.